File: Practice/Apples and Oranges/cpp/appleorange.cpp
/*
* Problem Link - https://www.hackerearth.com/problem/algorithm/problem-2-10/
*/


#include<iostream>
using namespace std;
int main() {
	int a, o, count, i;
	long int big;
	cin >> a >> o;
	if (a > o) {
		big = a;
	} else {
		big = o;
	}
	if (a < 1000000000 && o < 1000000000) {
		for (i = 1; i <= big; i++) {
			if (a % i == 0 && o % i == 0) {
				count = i;
			}
		}
		cout << a / count << " " << o / count;
	}
	return 0;
}

File: Practice/Apples and Oranges/java/AppleOrange.java
/*
* Problem Link - https://www.hackerearth.com/problem/algorithm/problem-2-10/
*/


import java.util.ArrayList;
import java.util.Scanner;
 
class AppleOrange {
    public static void main(String args[] ) throws Exception {
        Scanner s = new Scanner(System.in);
		int apples = s.nextInt();
		int oranges = s.nextInt();
		ArrayList<Integer> al = new ArrayList<Integer>();
		for (int i = 1; i <= 12000; i++) {
			if (apples % i == 0 && oranges % i == 0) {
				al.add(i);
			}
			continue;
		}
		int d = al.get(al.size() - 1);
		System.out.println(apples / d + " " + oranges / d);
		s.close();
	}
}